packages/core/src/anchor/ethereum/ethereum-anchor-validator.ts

```diff
@@ -171,34 +172,42 @@ export class EthereumAnchorValidator implements AnchorValidator {
       // determine network based on a chain ID
 
       const provider: providers.BaseProvider = this._getEthProvider(chainId)
-      const transaction: TransactionResponse = await this._getTransaction(provider, txHash)
-
-      if (!transaction) {
-        if (!this.ethereumRpcEndpoint) {
-          throw new Error(
-            `Failed to load transaction data for transaction ${txHash}. Try providing an ethereum rpc endpoint`
-          )
-        } else {
-          throw new Error(`Failed to load transaction data for transaction ${txHash}`)
+      const transaction: TransactionResponse = await backOff(async () => {
+        const tx = await this._getTransaction(provider, txHash)
+        if (!tx) {
+          if (!this.ethereumRpcEndpoint) {
+            throw new Error(
```

Comment:
how does the `backOff` function handle if an error is thrown?  Ideally we want to log these errors when they are thrown even if it succeeds on retry